crates/trigger/src/cli.rs

```diff
@@ -54,6 +55,13 @@ pub struct FactorsTriggerCommand<T: Trigger<B::Factors>, B: RuntimeFactorsBuilde
     )]
     pub log: Option<PathBuf>,
 
+    /// If set, Spin deletes the log files before starting the application.
+    #[clap(
+        name = SPIN_REFRESH_LOGS,
+        long = "refresh-logs",
```

Comment:
I'd really like to align on the language here.  Is the consensus for "refresh logs"?  It feels ambiguous to me and I'd prefer "clear logs" or "truncate logs" or something.  (But if we do make a change then the language needs to line up everywhere, not just on this line - field names, EVs, docs, etc.)